### ISSUE TITLE
Zimbabwe flag shows for a second before the geoIpLookup is returned

### ISSUE THREAD

**Issue by gobruins2154** [Bluefieldscom/intl-tel-input#245]
Hi, 

Everything works fine if I just set the default to "us", but I really like the geoIpLookup feature but the Zimbabwe flag shows for 1 second before setting the correct flag. I am using the latest 6.0 release from: 
https://github.com/Bluefieldscom/intl-tel-input/blob/master/build/js/intlTelInput.min.js
https://github.com/Bluefieldscom/intl-tel-input/blob/master/build/css/intlTelInput.css
https://github.com/Bluefieldscom/intl-tel-input/blob/master/build/img/flags.png

If I remove the geoIpLookup function and don't set a default, the Zimbabwe flag shows up and doesn't get changed.

This is what my code looks like:

```javascript
 $("#sms_to_download_mobile_phone").intlTelInput({
        //autoFormat: false,
        //autoHideDialCode: false,
        //defaultCountry: "in",
        //nationalMode: true,
        //numberType: "MOBILE",
        //onlyCountries: ['us', 'gb', 'ch', 'ca', 'do'],
        //preferredCountries: ['cn', 'jp'],
        //responsiveDropdown: true,
        preferredCountries: ["us", "gb", "au", "ca"],
        defaultCountry: "auto",
        geoIpLookup: function(callback) {

            $.get('https://www.telize.com/geoip').always(function(json) {
                var countryCode = (json && json.country_code) ? json.country_code.toLowerCase() : "us";
                callback(countryCode);
            });

        },
        utilsScript: "<%=javascript_path "vendor/utils.js" %>",
        responsiveDropdown: true
    });

```

Great open source tool! So helpful!

**Comment by jackocnr** [comment]
Thanks for reporting this. Not sure when this was introduced, but I just fixed it in v6.0.1. Please let me know if you have any other problems.

**Comment by gobruins2154** [comment]
Thanks, that fixed it! Love the component.
